src/Compilers/Core/Portable/SymbolDisplay/SymbolDisplayMiscellaneousOptions.cs

```diff
@@ -61,5 +61,10 @@ public enum SymbolDisplayMiscellaneousOptions
         /// (Review design for this option before shipping. See https://github.com/dotnet/roslyn/issues/26198)
         /// </summary>
         IncludeNullableReferenceTypeModifier = 1 << 6,
+
+        /// <summary>
+        /// Allow the use of <c>default</c> instead of <c>default(T)</c> where applicable.
+        /// </summary>
+        AllowDefaultLiteral = 1 << 7,
```

Comment:
should this be under SymbolDisplayMemberOptions not misc options?

Reply:
I'll leave it here unless @dotnet/roslyn-compiler requests a move.